spec/models/picture_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Picture, type: :model do
+  it { should have_many :comments }
+  it { should belong_to :user}
+  it { should have_many :likes }
+
+  describe 'comments' do
+    describe 'build_with_user' do
+      let(:user) { User.create email: 'test@test.com' }
+      let(:picture) { Picture.create title: 'Test' }
+      let(:comment_params) { {words: "test"} }
+
+      subject(:comment) { picture.comments.build_with_user(comment_params, user) }
```

Comment:
You're testing here an ActiveRecord method, which you don't actually need to do, you should assume ActiveRecord has tested this for you. What you really wanted here was a custom method in your picture model `picture.create_comment` or something and then test what you want that method to do